torch/fx/passes/utils/matcher_utils.py

```diff
@@ -0,0 +1,230 @@
+from dataclasses import dataclass, field
+from collections import defaultdict
+import copy
+from torch.fx.graph import Graph
+from torch.fx.node import Node
+from torch.fx._compatibility import compatibility
+from typing import Dict, List, Set
+
+__all__ = ['SubgraphMatcher']
+
+
+@compatibility(is_backward_compatible=False)
```

Comment:
You don't need this if the class name starts with `_`, since it's a private API

Reply:
fixed.